docs/_include/variable_hydro.rst

```diff
@@ -0,0 +1,113 @@
+To turn on variable hydrodynamics, use the ``body.variableHydro.option`` flag in the 
+wecSimInputFile and input multiple H5 files during the body initialization::
+
+    body(1) = bodyClass({'H5FILE_1.h5','H5FILE_2.h5','H5FILE_3.h5','H5FILE_4.h5','H5FILE_5.h5');
+    body(1).variableHydro.option = 1;
+
+
+If ``body.variableHydro.option=0`` any extra H5 files in the body initialization 
+are ignored. If only one H5 file is submitted to a file, variable hydrodynamics 
+are turned off.
+
+The flag ``body.variableHydro.hydroForceIndexInitial`` allows the user to set the
+default index used to define the hydrodynamic coefficients. This initial dataset
+should represent the body at the start of a simulation and likely its equilibrium 
+position. The initial dataset will be used to define the mass, center of gravity, 
+and center of buoyancy for a simulation.
+
+This parameter is flexible because an initial index of zero is not always convenient
+and can complicate indexing logic. For example, consider a flap-based device with
+multiple hydrodynamic datasets at various pitch angles (-50:10:50). It is most convenient
+to treat these angles in numerical order in BEM simulations, indexing logic, and other 
+data processing. In this case the initial position is at a pitch angle of 0 so ``body.variableHydro.hydroForceIndexInitial=6;``.
+
+Compatibility
+"""""""""""""
+Variable hydrodynamics is not compatible with certain WEC-Sim features. It cannot be used concurrently with:
+
+* State-space radiation calculations
+* FIR Filter radiation calculations
+* Generalized body modes within the same body. Seperate bodies could each have generalized body modes and variable hydrodynamics however.
+* Non-hydrodynamic and drag bodies
+
+
+Application
+""""""""""""
+See the :ref:`user-applications-variable-hydro` WEC-Sim_Application for a demonstration of setting up and using variable hydrodynamics.
+
+Tips
+""""
+* Investigate which advanced feature (variable hydrodynamics, passive yaw, large XY displacements, etc) will accomplish your modeling goals most effectively
+* Keep the state's range small
+* Input BEM data to cover the entire range of the state
+* A very fine state discretization may be required. Conduct a state discretization study to find the resolution required for accurate results.
+* If a very fine state discretization is required, use may be able to use BEMIO to create new datasets by interpolating between BEM simulations instead of simulating thousands of distinct BEM solutions. See the :ref:`user-applications-variable-hydro` Application for an example.
+* Validate using high fidelity data
+* The hydroData directory may become very large
+* All BEM data is contained within the ``body`` variable. Pre-processing remains very fast, so it's not recommended to save ``body`` to an output file or the file size may increase drastically.
+* Conditions that require a varying mass, center of gravity, or center of buoyancy are not yet implemented.
```

Comment:
Similar comment, I this this is better suited as a note that this feature should be used with extreme caution. I also recommend moving a lot of these recommendations to the README in the variable hydro application case.

Reply:
I can change this to a note or paragraph, but I don't think the notes will be read if moved to the Application readme. Those documents tend to contain very basic information about a specific case, whereas these tips will be applicable to any simulation with variable hydrodynamics